src/status_im/contexts/chat/messenger/messages/view.cljs

```diff
@@ -19,57 +19,50 @@
+        chat-list-scroll-y       (reanimated/use-shared-value 0)
         alert-banners-top-margin (rf/sub [:alert-banners/top-margin])
-        chat-exist?              (rf/sub [:chats/current-chat-exist?])]
-    (when chat-exist?
-      [rn/keyboard-avoiding-view
-       {:style                    (style/keyboard-avoiding-container theme)
-        :keyboard-vertical-offset (- (if platform/ios? alert-banners-top-margin 0) (:bottom insets))}
-       [:<>
-        [list.view/messages-list-content props]
-        [scroll-to-bottom/button props]]
-       [messages.navigation/view props]
-       [footer props]])))
-
-(defn lazy-chat-screen
-  [chat-screen-layout-calculations-complete? *screen-loaded?*]
-  (let [screen-loaded?         (rf/sub [:shell/chat-screen-loaded?])
-        distance-from-list-top (reanimated/use-shared-value 0)
-        chat-list-scroll-y     (reanimated/use-shared-value 0)
-        props                  {:insets (safe-area/get-insets)
-                                :content-height (atom 0)
-                                :layout-height (atom 0)
-                                :distance-atom (atom 0)
-                                :distance-from-list-top distance-from-list-top
-                                :chat-list-scroll-y chat-list-scroll-y
-                                :chat-screen-layout-calculations-complete?
-                                chat-screen-layout-calculations-complete?}]
-    (when *screen-loaded?*
-      (rn/use-mount #(reset! *screen-loaded?* true)))
-    (when-not (if *screen-loaded?* @*screen-loaded?* screen-loaded?)
-      (reanimated/set-shared-value chat-screen-layout-calculations-complete? false)
-      (reanimated/set-shared-value distance-from-list-top 0)
-      (reanimated/set-shared-value chat-list-scroll-y 0))
-    (when (if *screen-loaded?* @*screen-loaded?* screen-loaded?)
-      [chat-screen props])))
+        insets                   (safe-area/get-insets)]
+    [rn/keyboard-avoiding-view
+     {:style                    (style/keyboard-avoiding-container theme)
+      :keyboard-vertical-offset (- (if platform/ios? alert-banners-top-margin 0)
+                                   (:bottom insets))}
+     [:<>
+      [list.view/messages-list-content
+       {:insets                 insets
+        :distance-from-list-top distance-from-list-top
+        :chat-list-scroll-y     chat-list-scroll-y
+        :layout-height          layout-height
+        :on-layout-done?        on-layout-done?}]
+      [scroll-to-bottom/button chat-list-scroll-y]]
+     [messages.navigation/view]
+     [footer layout-height]]))
 
 (defn chat
   []
-  (let [chat-screen-layout-calculations-complete? (reanimated/use-shared-value false)
-        jump-to-enabled?                          (ff/enabled? ::ff/shell.jump-to)
-        *screen-loaded?*                          (when-not jump-to-enabled?
-                                                    (reagent/atom false))]
```

Comment:
`ratom` being created in a non-form-2-component. This works because we aren't re-rendering this component, but as soon as `ff/enabled?` changes during this screen mounted, it won't work.

Although it works now, I prefered to refactor to properly create the ratom.